docs/en/getting-started/install.md

```diff
@@ -38,31 +38,43 @@ On Linux, macOS and FreeBSD:
   curl https://clickhouse.com/ | sh
   ```
 
-1. Run the following command to start the ClickHouse server:
+2. Run the following command to start [clickhouse-local](../operations/utilities/clickhouse-local.md):
 
     ```bash
-    ./clickhouse server
+    ./clickhouse
     ```
 
-   The first time you run this script, the necessary files and folders are created in the current directory, then the server starts.
+   Clickhouse-local allows you to process local and remote files using ClickHouse's powerful SQL and without a need for configuration. Table
```

Comment:
Maybe `Clickhouse-local` should start with the lowercase and be inside backticks to make it look ok?

Reply:
Good idea, done.